source/solutions-library/claim-management.txt

```diff
@@ -87,67 +91,121 @@ context.
+
+For detailed setup instructions, follow the README of `this
+Github repository <https://github.com/mongodb-industry-solutions/RAG-Insurance>`__.
 You’ll be guided through the following steps:
 
-- AWS Account with Bedrock access
-- Atlas connection setup
-- Dataset download
-- LLM configuration options
-- Vector Search index creation
+To build the solution, follow these instructions:
 
-Visit the `Atlas Vector Search Quick Start guide
-<https://www.mongodb.com/docs/atlas/atlas-vector-search/tutorials/vector-search-quick-start/?tck=ai_as_web>`__
-to try our semantic search tool now.
+.. procedure::
+   :style: normal
+
+   .. step:: Set up MongoDB database and collections
+
+      Create a new database in MongoDB Atlas called ``demo_rag_insurance`` and
+      use the provided dataset ``demo_rag_insurance_claims.json`` to create a
+      collection called ``claims_final``.
+
+   .. step:: Create a Vector Search Index
 
-This `document
-<https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-type/#create-an-atlas-vector-search-index>`__
-walks you through the creation and configuration of the Vector Search
-index. Make sure you follow this structure:
-
-.. code-block:: json
-   :copyable: true 
-
-   {
-      "fields": [
-         { 
-            "type": "vector", 
-            "path": "claimDescriptionEmbedding",
-            "numDimensions": 350, 
-            "similarity": "cosine"
+      `Create and configure an Atlas Vector Search Index
+      <https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-type/#create-an-atlas-vector-search-index>`__
+      for ``claimDescriptionEmbeddingCohere`` called
+      ``vector_index_claim_description_cohere``. You must structure the search index as follows:
+
+      .. code-block:: json
+         :copyable: true 
+
+         {
+            "fields": [
+               { 
+                  "type": "vector", 
+                  "path": "claimDescriptionEmbeddingCohere",
+                  "numDimensions": 350, 
+                  "similarity": "cosine"
+               }
+            ]
          }
-      ]
-   }
 
-Ultimately, you have to run both the front and the back end. You’ll
-access a web UI that allows you to ask questions to the LLM, obtain an
-answer, and see the reference documents used as context.
+   .. step:: Configure the Backend
+      
+      Set up a virtual environment using Poetry.
+
+   .. step:: Interact with the API
+
+      Start the backend server. 
+
+   .. step:: Connect to the Frontend
+
+      Configure environment variables and run the frontend.
+
+Ultimately, you have to run both the front and back end. You’ll access a web UI
+that allows you to ask questions to the LLM, obtain an answer, and see the
+reference documents used as context.
+
+Visit the `Atlas Vector Search Quick Start guide
+<https://www.mongodb.com/docs/atlas/atlas-vector-search/tutorials/vector-search-quick-start/?tck=ai_as_web>`__
+to try our semantic search tool now.
```

Comment:
Also ref here. And i think(?) we like to structure sentences as "to do A, visit B."
```suggestion
To try MongoDB's semantic search tool now, 
visit the :ref:`Atlas Vector Search Quick Start guide <vector-search-quick-start>`.
```